conf/messages.en

```diff
@@ -855,14 +855,15 @@ cds.subscription-details.amend.legal-entity.user.header=Edit legal entity
 cds.subscription-details.amend.legal-entity.submitButton=Save and go to dashboard
 cds.multi.input.add-another=Add another
 #Subscription VAT details
-cds.subscription.vat-details.heading=What are your UK VAT details?
-cds.subscription.vat-details.intro-text=You can find these details on your VAT registration certificate.
-cds.subscription.vat-details.postcode=The postcode of your VAT registration address
+cds.subscription.vat-details.heading=Your UK VAT details
+cds.subscription.vat-details.intro-text=You can find your VAT details on your VAT registration certificate. To view this login to your tax account and select ''More VAT details''. Select the ''View VAT certificate'' link under the heading ''Help with tax''.
+cds.subscription.vat-details.intro-text.link=Find your VAT details (opens in new tab).
+cds.subscription.vat-details.postcode=VAT registration address postcode
 cds.subscription.vat-details.postcode.required.error=Enter a valid postcode of your VAT registration address
 cds.subscription.vat-details.postcode.length.error=The postcode must be a maximum of 8 characters
 cds.subscription.vat-details.vat-number=VAT number
 cds.subscription.vat-details.vat-effective-date=Effective VAT date
-cds.subscription.vat-details.vat-effective-date.hint=For example, 31 3 1980
+cds.subscription.vat-details.vat-effective-date.hint=For example, 01 11 2007
```

Comment:
do we need to change the vat effective date for cy as well

Reply:
I have added the date to welsh messages as well now.